tests/util.py

```diff
@@ -357,6 +359,33 @@ def _perform_request(self, path, **kwargs):
 
         return self.client.open(path, **kwargs)
 
+    def sort_inner_lists(self, obj):
+        """
+        Sort all inner lists in LoRa objects ascending by the 'from' date of
+        their elements -- e.g. The list located at
+        obj['relationer']['enhedstype']
+
+        This is purely to help comparison tests, as we don't care about the
+        list ordering
+        """
+        obj = copy.deepcopy(obj)
+        for outer_value in filter(lambda x: type(x) is dict, obj.values()):
+            for inner_key in outer_value:
+                outer_value[inner_key] = sorted(outer_value[inner_key],
+                                                key=common.get_effect_from)
+        return obj
+
+    def assertEqualLoRa(self, expected, actual):
```

Comment:
“LoRA” isn't terribly helpful in this regard — perhaps `assertRegistrationsEqual` instead? (That appears to be what you're actually want to compare.) Also, a docstring would be nice.

Reply:
Yeah, I'm terrible at naming things. I'll fix it.